### PR TITLE
Add WMX - WMIX DISTRIBUIDORA - Update facilities.json

### PR DESCRIPTION
**Pull request by japierce1028**
Opt IN

### CODE REVIEW

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -7089,6 +7089,16 @@
     "code": "WMS",
     "description": "WILD MIKE STUDIO"
   },
+  {
+    "code": "WMX",
+    "contact": {
+      "address": "Av. Pres. Kennedy, 569 - 902 - Boqueirão, Praia Grande - SP, 11701-335, Brazil",
```
Comment (palemieux) [inline src/main/data/facilities.json:7095]: Is 902 an apartment number?

If so, should the address be formatted as follows:

Av. Pres. Kennedy, 569, apt. 902, Boqueirão, Praia Grande - SP, CEP 11701-335, Brazil

**Review by palemieux — CHANGES_REQUESTED** [review]

File: src/main/data/facilities.json (hunk 1 above)
Comment (japierce1028) [reply src/main/data/facilities.json:7095]: The actual entry was: "Av: Presidente Kennedy, 569, sala 902" and "sala" means "room" in Brazil.  So I'm not sure it is an apartment number. Maybe "room" 902?

Comment (palemieux) [reply src/main/data/facilities.json:7095]: I think we should leave _sala_.

[the remark this answers is not in the document]
Comment (japierce1028) [reply src/main/data/facilities.json]: I think the current submission has Sala in it... I think it's ready to go (although your suggestion has an extra comma).

**Review by palemieux — APPROVED** [review]

**Pull request merged**